leshan-core/src/test/java/org/eclipse/leshan/core/node/codec/LwM2mNodeEncoderTest.java

```diff
@@ -293,7 +293,7 @@ public void json_encode_timestamped_Object() throws CodecException {
         b.append("{\"n\":\"0/1\",\"v\":22.9,\"t\":210},");
         b.append("{\"n\":\"0/0\",\"sv\":\"a string\",\"t\":220},");
         b.append("{\"n\":\"0/1\",\"v\":22.4,\"t\":220},");
-        b.append("{\"n\":\"1/1\",\"v\":23,\"t\":220},");
+        b.append("{\"n\":\"1/1\",\"v\":23.0,\"t\":220},");
```

Comment:
This seems to be how jackson serializes double. Valid json and one could argue double is better to have decimal point.

I don't think is worrying just an effect of different library.